snews_cs/snews_coinc.py

```diff
@@ -76,140 +61,95 @@ def reset_df(self):
         """ Resets coincidence arrays if coincidence is broken
 
         """
-        if self.coinc_broken:
-            del self.cache_df
-            self.cache_df = pd.DataFrame()
-        else:
-            pass
-
-    # ------------------------------------------------------------------------------------------------------------------
-    def reset_cache(self):
-        """ Resets mongo cache and all coincidence arrays if coincidence is broken
-
-        """
-        if self.coinc_broken:
-            self.reset_df()
-            # self.storage.purge_cache(coll='CoincidenceTier')
-            self.coinc_broken = False
-            self.delta_t = None
-            self.cache_reset = True
-        else:
-            pass
-
-    # ------------------------------------------------------------------------------------------------------------------
-    def copy_row_and_append(self, ind, delta_t, sub_list_num):
-        duplicate_row = self.cache_df.iloc[ind].copy(deep=False)
-        duplicate_row['nu_delta_t'] = np.abs(delta_t)
-        duplicate_row['sub_list_num'] = sub_list_num
-        return self.cache_df.append(duplicate_row, ignore_index=True)
+        del self.cache_df
+        self.cache_df = pd.DataFrame(columns = self.column_names)
+        self.initial_set = False
 
     # ------------------------------------------------------------------------------------------------------------------
     def coincident_with_whole_list(self, message, sub_list_num):
-        message_nu_time = self.times.str_to_hr(message['neutrino_time'])
+        """
```

Comment:
In this function, I circled back to your flag actually <br> 
I return `self.in_coincidence = True` for everything `'COINCIDENT'` and `'EARLY_COINCIDENT'` <br>
for `NOT_COINCIDENT` it remains `False` <br>
However, as it creates a new sublist if there is nothing `self.in_coincidence`, I return `True` also for `'ALREADY_IN_LIST'` but later on line 141, I only submit alert if it was not already in list